test/javascripts/acceptance/assigned-topic-test.js

```diff
@@ -89,6 +90,7 @@ acceptance("Discourse Assign | Assigned topic", function (needs) {
     tagging_enabled: true,
     assigns_user_url_path: "/",
     assigns_public: true,
+    enable_assign_status: true,
```

Comment:
We should have tests that check the status dropdown is present (or not) depending on when this setting is enabled or not, right?